bundle/transform/profile/profiles.yml

```
@@ -7,21 +7,21 @@ meltano:
     snowflake:
       type: snowflake
       threads: 2
-      account: "{{ env_var('SF_ACCOUNT') }}"
-      user: "{{ env_var('SF_USER') }}"
-      password: "{{ env_var('SF_PASSWORD') }}"
-      role: "{{ env_var('SF_ROLE') }}"
-      database: "{{ env_var('SF_DATABASE') }}"
-      warehouse: "{{ env_var('SF_WAREHOUSE') }}"
+      account: "{{ env_var('TARGET_SNOWFLAKE_ACCOUNT') }}"
+      user: "{{ env_var('TARGET_SNOWFLAKE_USER') }}"
+      password: "{{ env_var('TARGET_SNOWFLAKE_PASSWORD') }}"
+      role: "{{ env_var('TARGET_SNOWFLAKE_ROLE') }}"
+      database: "{{ env_var('TARGET_SNOWFLAKE_DATABASE') }}"
```

I think this should say  `TARGET_SNOWFLAKE_DBNAME` like in https://github.com/meltano/files-dbt/pull/16, as that's the setting name the default `target-snowflake` variant `transferwise` uses.

```suggestion
      database: "{{ env_var('TARGET_SNOWFLAKE_DBNAME }}"
```